tests/README.md

```diff
@@ -0,0 +1,124 @@
+perf_bench/viper_call1c.py: 20849.88 5.8783 14444.80 6.6295
+perf_bench/viper_call2a.py: 16156.25 11.2956 18818.59 11.7959
+perf_bench/viper_call2b.py: 22047.38 8.9484 13725.73 9.6800
+```
+
+The numbers across each line are times and scores for the test:
+
+* Runtime average (microseconds, lower is better)
+* Runtime standard deviation as a percentage
+* Score average (units depend on the benchmark, higher is better)
+* Score standard deviation as a percentage
+
+### Comparing performance
+
+Usually you want to know if something is faster or slower than a reference. To do this, copy the output of each `run-perfbench.py` run to a text file.
+
+This can be done multiple ways, but one way on Linux/macOS is with the `tee` utility: `./run-perfbench.py 2000 10000 | tee host-run1.txt`
+
+Once you have two files with output from two different runs (maybe with different code or configuration), compare the runtimes with `./run-perfbench.py -t host-run1.txt host-run2.txt` or compare scores with `./run-perfbench.py -s host-run1.txt host-run2.txt`:
+
+```
+> ./run-perfbench.py -s host-run1.txt host-run2.txt
+N=2000 M=10000             host-run1.txt -> host-run2.txt         diff      diff% (error%)
+bm_fannkuch.py                  87.36 ->      80.93 :      -6.43 =  -7.360% (+/-8.50%)
+bm_fft.py                   145140.65 ->  128883.84 :  -16256.81 = -11.201% (+/-19.10%)
+bm_float.py                 312261.93 ->  335495.81 :  +23233.88 =  +7.441% (+/-25.24%)
+bm_hexiom.py                   733.49 ->     727.74 :      -5.75 =  -0.784% (+/-8.22%)
+bm_nqueens.py               242657.95 ->  240108.78 :   -2549.17 =  -1.051% (+/-11.51%)
+bm_pidigits.py                7989.53 ->    7860.92 :    -128.61 =  -1.610% (+/-9.22%)
+core_import_mpy_multi.py     42024.81 ->   31954.13 :  -10070.68 = -23.964% (+/-27.60%)
+core_import_mpy_single.py     4001.31 ->    3637.18 :    -364.13 =  -9.100% (+/-24.56%)
+core_qstr.py                  8159.22 ->    7795.84 :    -363.38 =  -4.454% (+/-13.77%)
+core_yield_from.py           12413.00 ->   13614.51 :   +1201.51 =  +9.679% (+/-11.70%)
+misc_aes.py                  17710.98 ->   17432.07 :    -278.91 =  -1.575% (+/-4.28%)
+misc_mandel.py              179709.35 ->  175101.68 :   -4607.67 =  -2.564% (+/-30.04%)
+misc_pystone.py              98301.22 ->  102387.50 :   +4086.28 =  +4.157% (+/-12.01%)
+misc_raytrace.py             26414.96 ->   27338.95 :    +923.99 =  +3.498% (+/-16.39%)
+viper_call0.py               25682.85 ->   19381.45 :   -6301.40 = -24.535% (+/-22.30%)
+viper_call1a.py              34739.62 ->   18510.85 :  -16228.77 = -46.715% (+/-3.50%)
+viper_call1b.py              27310.98 ->   15157.11 :  -12153.87 = -44.502% (+/-7.39%)
+viper_call1c.py              18709.36 ->   14638.61 :   -4070.75 = -21.758% (+/-32.15%)
+viper_call2a.py              18521.96 ->   19328.71 :    +806.75 =  +4.356% (+/-11.48%)
+viper_call2b.py              21312.76 ->   12458.97 :   -8853.79 = -41.542% (+/-24.09%)
```

Comment:
These percentage errors are very large (that's common when run on a PC), so this diff is not very useful.  Maybe do a run on PYBv1.1 instead?

Maybe also add a comment at the bottom here that you should always check the error% is relatively low, so the numbers can be trusted.

Reply:
Done, thanks for picking me up on being lazy here (I picked host builds as the easiest way to see some differences!) :)